### PR TITLE
task solution

### PR DESCRIPTION
**Pull request by vladyslav-harkusha**
[DEMO LINK](https://vladyslav-harkusha.github.io/react_dynamic-list-of-goods/)

### CODE REVIEW

File: src/api/goods.ts [hunk 1]
```diff
@@ -3,17 +3,27 @@ import { Good } from '../types/Good';
...
 };
 
-export const getRedGoods = () => {
-  return getAll()
-    .then(goods => goods); // get only red
+export const get5First = async () => {
+  const goods = await getAll();
+
+  return goods
+    .sort((g1, g2) => (g1.name.localeCompare(g2.name)))
```
Comment (OleksandrGlulhovskiy) [inline src/api/goods.ts:21]: it's better to named it firstGood and secondGood or a and b

File: src/App.scss [hunk 2]
```diff
@@ -1 +1,60 @@
...
+      background-color: rgb(222, 252, 245);
+      cursor: pointer;
+    }
+
+    &:focus {
+      background-color: rgb(128, 255, 225);
+      color: white;
+    }
+
+    &--five {
```
Comment (OleksandrGlulhovskiy) [inline src/App.scss:32]: Perfect

**Review by OleksandrGlulhovskiy — COMMENTED** [review]
Good Job)

**Review by sergii-nosachenko — APPROVED** [review]
Good work, nice style 🚀🚀🚀

**Pull request closed without merging**